.gitignore

```diff
@@ -10,3 +10,4 @@ pubspec.lock
 # Directory created by dartdoc
 doc/api/
 
+/.idea/
```

Comment:
Let's add this in a separate PR.

Reply:
done, https://github.com/grpc/grpc-dart/pull/251